packages/trader/src/AppV2/Components/TradeParameters/Barrier/barrier.tsx

```diff
@@ -49,7 +50,11 @@ const Barrier = observer(({ is_minimized }: TDurationProps) => {
                 readOnly
                 label={<Localize i18n_default_text='Barrier' key={`barrier${is_minimized ? '-minimized' : ''}`} />}
                 value={v2_params_initial_values.barrier_1 || barrier_1}
-                onClick={() => setIsOpen(true)}
+                onClick={(e: React.MouseEvent<HTMLInputElement, MouseEvent>) => {
+                    removeFocus(e);
+                    setIsOpen(true);
+                }}
+                onMouseDown={removeFocus}
```

Comment:
On normal devices `removeFocus` inside of `onClick` works as expected. But on IOS `blur()` inside of onClick is not working. During digging IOS documentation and devs forums, I tried different ways (touch events, re-applying focus on hidden elements, css tricks), but finally found the solution with mouse down event listener. 
So `onMouseDown` stands for IOS